### PR TITLE
Examples quaternion

### PR DESCRIPTION
**Pull request by mbouron**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -21,6 +21,8 @@ Versioning](https://semver.org/spec/v2.0.0.html) for `libnopegl`.
   information (types, choices, etc)
 - The documentation has been reworked, notably to include code examples with
   associated rendering and graph
+- `Transform` node now accept a node as input parameter and can be animated
```
Comment (ubitux) [inline CHANGELOG.md:24]: accepts

File: libnopegl/src/node_transform.c [hunk 2]
```diff
@@ -68,8 +69,19 @@ static int transform_init(struct ngl_node *node)
...
+    struct transform_opts *o = node->opts;
+
+    int ret = ngli_node_update_children(node, t);
+    if (ret < 0)
+        return ret;
+
+    if (o->matrix_node) {
+        float *data = ngli_node_get_data_ptr(o->matrix_node, o->matrix);
+        memcpy(s->trf.matrix, data, sizeof(s->trf.matrix));
+    }
```
Comment (ubitux) [inline libnopegl/src/node_transform.c:82]: Don't you need to update the matrix first, then the update of the child?

File: CHANGELOG.md (hunk 1 above)
Comment (mbouron) [reply CHANGELOG.md:24]: Done.

File: libnopegl/src/node_transform.c (hunk 2 above)
Comment (mbouron) [reply libnopegl/src/node_transform.c:82]: In the update phase, the children must be updated first so the parents can rely on them.

**Review by ubitux — APPROVED** [review]

**Pull request merged**